src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
no need to check that `days >= 3` because we would have already returned the value if the condition was true in the prev `if`

Reply:
I fix it, can you check it again?